src/polycubed/src/cube.cpp

```diff
@@ -39,8 +39,10 @@ Cube::Cube(const std::string &name, const std::string &service_name,
   forward_chain_ = std::unique_ptr<ebpf::BPFArrayTable<uint32_t>>(
       new ebpf::BPFArrayTable<uint32_t>(forward_));
 
-  // add free ports
-  for (uint16_t i = 0; i < _POLYCUBE_MAX_PORTS; i++)
+  /* add free ports, port id 0 is reserved by pcn-iptables as wild card, so
+   * it won't be put into the free pool
+   */
+  for (uint16_t i = 1; i < _POLYCUBE_MAX_PORTS+1; i++)
```

Comment:
@mauriciovasquezbernal Are we safe here? We're cycling till _POLYCUBE_MAX_PORTS+1, while internal variables (e.g., BPF_TABLE_SHARED("array", int, u32, forward_chain_, _POLYCUBE_MAX_PORTS); ) are dimensioned with _POLYCUBE_MAX_PORTS.

Reply:
It's not safe. If a cube gets the `_POLYCUBE_MAX_PORTS` port id it'll be **not** able to use that port because the lookup in that map will fail.